src/test/core/aztec/decoder/Decoder.spec.ts

```diff
@@ -167,14 +163,18 @@ describe('DecoderTest', () => {
         let bool1: boolean[] = [true];
         let bool7: boolean[] = [true, false, true, false, true, false, true];
         let bool8: boolean[] = [true, false, true, false, true, false, true, false];
-        let bool9: boolean[] = [true, false, true, false, true, false, true, false, true];
-        let bool16: boolean[] = [false, true, true, false, false, false, true, true, true, true, false, false, false, false, false, true];
-        let byte0: /*byte[]*/ Uint8Array = new Uint8Array([]);
-        let byte1: /*byte[]*/ Uint8Array = new Uint8Array([-128]);
-        let byte7: /*byte[]*/ Uint8Array = new Uint8Array([-86]);
-        let byte8: /*byte[]*/ Uint8Array = new Uint8Array([-86]);
-        let byte9: /*byte[]*/ Uint8Array = new Uint8Array([-86, -128]);
-        let byte16: /*byte[]*/ Uint8Array = new Uint8Array([99, -63]);
+        let bool9: boolean[] = [
+            true, false, true, false, true, false, true, false,
+            true];
+        let bool16: boolean[] = [
+            false, true, true, false, false, false, true, true,
+            true, true, false, false, false, false, false, true];
```

Comment:
I couldn't understand the logic behind these new linebreaks around the code. Did you do these line breaks or was it a result from tslint or prettier-like tools? Just curious.